h/assets.py

```diff
@@ -147,3 +151,5 @@ def includeme(config):
     # integration can be configured in app.py
     config.registry['assets_env'] = assets_env
     config.registry['assets_client_env'] = assets_client_env
+
+    config.add_request_method(assets_env.asset_url, 'asset_url')
```

Comment:
These asset urls are used only in the templates so perhaps a neater way of exposing this function there would be to follow the pattern in [app.py](https://github.com/hypothesis/h/blob/f671cedcc615c0b55e7c21a7c5fe63fe7051c7fc/h/app.py#L20) and add the `asset_url` function as a Jinja2 global.